packages/gestalt/src/Button/__tests__/Button.test.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import React from 'react';
+import { create } from 'react-test-renderer';
+import { shallow } from 'enzyme';
+import Button from '../Button';
+
+test('<Button color="transparent" />', () => {
+  const tree = create(
+    <Button color="transparent" text="Hello World" />
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+test('Button handles click', () => {
+  const mockOnClick = jest.fn();
+  const wrapper = shallow(<Button text="Text" onClick={mockOnClick} />);
```

Comment:
Don't need to use a `.jsdom.test.js` file because the `shallow` render works in both and we can still find the native `<button />` element just fine